doc/issues.md

```diff
@@ -0,0 +1,26 @@
+= Current issue organization
+
+== Pipelines 
+
+This repo is using default zenhub pipelines (if ZenHub is installed): 
+
+- New Issues - need to be triaged in x days
+- InProgress - someone assigned to the issue and actively working on it. 
```

Comment:
this would work better as a table IMO.

I think you can do this via something like:

```
| --- | --- |
| Issue State | Requirements |
| --- | --- |
| New Issues | Should be triaged in X days |
| --- | --- |
```